composer/loggers/mlflow_logger.py

```diff
@@ -322,7 +322,13 @@ def init(self, state: State, logger: Logger) -> None:
             self.run_name += f'-rank{dist.get_global_rank()}'
 
         # Register the global exception handler so that uncaught exception is tracked.
-        sys.excepthook = self._global_exception_handler
+        original_excepthook = sys.excepthook
```

Comment:
alternatively you could save `original_excepthook` on self (i.e. `self.original_excepthook`), so you don't need to use a lambda below

Reply:
yeah i think lambda is ok. could've also done functools.partial. don't feel particularly strongly between any of those options